ViaEventAssociation.Core.Domain/Aggregates/Requests/ValueObjects/AttendanceReason.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections;
+using ViaEventAssociation.Core.Tools.OperationResult;
+
+namespace ViaEventAssociation.Core.Domain.Aggregates.Requests.ValueObjects;
+
+public class AttendanceReason
+{
+    public string Reason { get; set; }
+
+    private AttendanceReason(string reason)
+    {
+        this.Reason = reason;
+    }
+
+    public static Result<AttendanceReason> Create(string reason)
+    {
+        var validate = Validate(reason);
+        return validate.IsSuccess
+            ? Result<AttendanceReason>.Success(new AttendanceReason(reason))
+            : Result<AttendanceReason>.Failure(validate.Errors.ToArray());
+    }
+
+    private static Result Validate(string reason)
+    {
+        var errors = new List<string>();
+        if (string.IsNullOrEmpty(reason))
```

Comment:
put every validation in the else block of this if statement if (string.IsNullOrEmpty(reason)). Otherwise, if reason will be null, the reason.Lenght validations will throw null pointer exceptions